QEfficient/src/common.py

```diff
@@ -83,4 +83,4 @@ def from_pretrained(cls, pretrained_model_name_or_path: str, *args, **kwargs) ->
         qeff_auto_model_class = MODEL_TYPE_TO_QEFF_AUTO_MODEL_MAP[model_type]
         assert issubclass(qeff_auto_model_class, QEFFBaseModel), f"Expected class that inherits {QEFFBaseModel}, got {type(qeff_auto_model_class)}"
 
-        return qeff_auto_model_class.from_pretrained(pretrained_model_name_or_path=pretrained_model_name_or_path)
+        return qeff_auto_model_class.from_pretrained(pretrained_model_name_or_path=pretrained_model_name_or_path, **kwargs)
```

Comment:
Do we want to pass `**kwargs` to the `AutoModelForCausalLM `__class__, might lead to hf_token issue in the hugginface.

Reply:
Good catch. Fixed